web/modules/custom/area_summary/src/Controller/AreaSummaryController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\area_summary\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Returns responses for area summary routes.
+ */
+class AreaSummaryController extends ControllerBase {
+
+  /**
+   * Builds the response.
+   */
+  public function build() {
+
+    $build['content'] = [
+      '#type' => 'item',
+      '#markup' => $this->t('It works!'),
```

Comment:
Is this on purpose?

Reply:
The controller wasn't needed to I've deleted it